### ISSUE TITLE
Add `node` alias to `stable` version

### ISSUE THREAD

**Issue by mrkrstphr** [vadimdemedes/trevor#43]
I have the following in my `.travis.yml` file:

```
node_js:
   - 6.0
   - node
```

`node` is listed as a "convenience version" in the [travis docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions) which will use the latest version of Node available. Similarly, there is `4`, `5`, and `6`, which alias the latest releases of those major versions.

Output:

```
❯ trevor

   ✔     6:  success
   ✖  node:  error
```

**Comment by gabrielcsapo** [comment]
This goes with #44 it is taking the version number and using that to pull down the appropriate node container.

**Comment by vadimdemedes** [comment]
Trevor supports `stable` alias, from the docs it looks like it's the same as `node`. I didn't even know `node` value existed, to be honest. I'll add an alias, since it's officially supported by Travis, thanks!

**Comment by vadimdemedes** [comment]
Fixed in `2.1.0` release.

**Comment by mrkrstphr** [comment]
Awesome, thanks!
